modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/AbstractDeploymentTestCase.java

```diff
@@ -1339,9 +1339,9 @@ protected void executeApplicationFlow(String flowName) throws Exception {
     executeApplicationFlow(flowName, null);
   }
 
-  protected CoreEvent executeApplicationFlow(String flowName, String correlationId) throws Exception {
```

Comment:
is this needed for this pr?

Reply:
nice, i changed this to core event and then reverted it because it didn't use it in the end. Apparently Sofia did the exact same changes and I brought them while rebasing to master and my local changes canceled them. 😮